packages/multichain/src/caip25Permission.test.ts

```diff
@@ -394,14 +395,7 @@ describe('caip25EndowmentBuilder', () => {
   });
 
   describe('permission validator', () => {
-    const findNetworkClientIdByChainId = jest.fn();
-    const listAccounts = jest.fn();
-    const { validator } = caip25EndowmentBuilder.specificationBuilder({
-      methodHooks: {
-        findNetworkClientIdByChainId,
-        listAccounts,
-      },
-    });
+    const { validator } = caip25EndowmentBuilder.specificationBuilder({});
```

Comment:
Nit: Strange that it lets you pass in an empty object here 🤔 It looks like it should take no parameters?

```suggestion
    const { validator } = caip25EndowmentBuilder.specificationBuilder();
```

Reply:
Ah, the type _does_ require an options object, but we have it typed as `Record<never, never>`. Probably we should revert my suggestion here for now, but ultimately I think we should consider updating the specification builder type to let us exclude the options completely.